packages/components/src/modifiers/hds-code-editor/themes/hds-dark-theme.ts

```diff
@@ -65,6 +74,141 @@ const hdsDark = EditorView.theme(
     '.cm-matchingBracket': {
       outline: `1px solid ${HDS_CODE_BLOCK_WHITE}`,
     },
+
+    // set a minimum height on the editor content when linting is enabled
+    '&.cm-lintingEnabled .cm-content': {
+      minHeight: '160px',
+    },
+
+    // replace the underline svg
+    '.cm-lintRange-error': {
+      backgroundImage: `url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="6" height="3">%3Cpath%20d%3D%22m0%202.5%20l2%20-1.5%20l1%200%20l2%201.5%20l1%200%22%20stroke%3D%22${encodeURIComponent(HDS_CODE_EDITOR_COLOR_FOREGROUND_CRITICAL)}%22%20fill%3D%22none%22%20stroke-width%3D%22.7%22%2F%3E</svg>')`,
+    },
+
+    // linting gutter
+    '&.cm-lintingEnabled .cm-lineNumbers .cm-gutterElement': {
+      paddingLeft: '8px',
+    },
+    '.cm-gutter-lint': {
+      width: '28px',
+    },
+    '.cm-gutter-lint .cm-gutterElement': {
+      padding: '0',
+    },
+    '.cm-lint-marker-error': {
+      width: '24px',
+      height: '24px',
+      padding: '6px',
+      content: `url('data:image/svg+xml,<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M9.56912 0.752289C8.69045 -0.126408 7.26581 -0.126415 6.38713 0.752272L0.730298 6.40911C-0.148388 7.28779 -0.148379 8.71243 0.730316 9.5911L6.38715 15.2478C7.26583 16.1265 8.69044 16.1265 9.56911 15.2478L15.2258 9.59109C16.1045 8.71241 16.1045 7.2878 15.2258 6.40912L9.56912 0.752289ZM5.21967 5.21955C5.51256 4.92665 5.98744 4.92665 6.28033 5.21955L8 6.93922L9.71967 5.21955C10.0126 4.92665 10.4874 4.92665 10.7803 5.21955C11.0732 5.51244 11.0732 5.98731 10.7803 6.28021L9.06066 7.99988L10.7803 9.71955C11.0732 10.0124 11.0732 10.4873 10.7803 10.7802C10.4874 11.0731 10.0126 11.0731 9.71967 10.7802L8 9.06054L6.28033 10.7802C5.98744 11.0731 5.51256 11.0731 5.21967 10.7802C4.92678 10.4873 4.92678 10.0124 5.21967 9.71955L6.93934 7.99988L5.21967 6.28021C4.92678 5.98731 4.92678 5.51244 5.21967 5.21955Z" fill="${encodeURIComponent(HDS_CODE_EDITOR_COLOR_FOREGROUND_CRITICAL)}"/></svg>');`,
+    },
+
+    // linter diagnostics panel
+    '.cm-panel.cm-panel-lint ul': {
+      backgroundColor: 'var(--token-color-palette-neutral-500)',
+      maxHeight: '80px',
```

Comment:
```suggestion
      maxHeight: '85px',
```
[Suggestion] A nit-picky design question but would it be possible to slightly increase this? With the current height one error is fully visible, and the second just peaks out, but not enough to really see that it's there. This isn't as much an issue when scrolling through the errors, but was noticeable when using the keyboard.

**Current**
<img width="733" alt="Screenshot 2025-03-20 at 11 44 34 AM" src="https://github.com/user-attachments/assets/37f3d57c-26d3-4c2f-b9fe-ae35a55cb473" />

**With 85px**
<img width="740" alt="Screenshot 2025-03-20 at 12 02 16 PM" src="https://github.com/user-attachments/assets/ebbcb96a-a7f2-456b-99a9-dead3ed39be5" />

Reply:
Ran it by lilith and implemented this